Doc/library/test.rst

```diff
@@ -1419,3 +1419,21 @@ script execution tests.
    containing the *source*.  If *compiled* is ``True``, both source files will
    be compiled and added to the zip package.  Return a tuple of the full zip
    path and the archive name for the zip file.
+
+.. function:: catch_unraisable_exception()
```

Comment:
This is placed under `script_helper` module doc. I Would prefer moving it `test.support` section. `.. versionadded:: 3.8` directive could be added to this.

Reply:
Oops, I moved the doc. I also added the markup.